app/src/marketplace/modules/dataUnion/services.js

```diff
@@ -27,9 +27,9 @@ export const getStreamrEngineAddresses = (): Array<string> => {
     return addresses
 }
 
-export const createJoinPartStream = async (productId: ProductId): Promise<Stream> => {
+export const createJoinPartStream = async (account: Address, productId: ProductId): Promise<Stream> => {
     const newStream: NewStream = {
-        id: `sandbox/dataunions/${productId}/joinPartStream`,
+        id: `${account}/dataunions/${productId}/joinPartStream`,
```

Comment:
Getting/setting permissions with `dataunions` in the path seems to throw an error in backend 🤔

Reply:
https://linear.app/streamr/issue/BACK-100/cannot-get-stream-permission-with-dataunion-in-path